fix: add string type to width and height of BaseTableProps

mengxi-ream opened the pull request:
<!--
First of all, thank you for your contribution! 😄
For requesting to pull a new feature or bugfix, please send it from a feature/bugfix branch based on the `main` branch.
Before submitting your pull request, please make sure the checklist below is confirmed.
Your pull requests will be merged after one of the collaborators approve.
Thank you!
-->

[[中文版模板 / Chinese template](https://github.com/VisActor/VTable/blob/main/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Refactoring
- [ ] Update dependency
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [x] Site / documentation update
- [ ] Demo update
- [ ] Workflow
- [ ] Chore
- [ ] Release
- [ ] Other (about what?)

### 🔗 Related issue link

<!--
1. Put the related issue or discussion links here.
2. close #xxxx or fix #xxxx for instance.
-->

### 💡 Background and solution

<!--
1. Describe the problem and the scenario.
2. GIF or snapshot should be provided if includes UI/interactive modification.
3. How to fix the problem, and list the final API implementation and usage sample if that is a new feature.
-->

### 📝 Changelog

<!--
Describe changes from the user side, and list all potential break changes or other risks.
--->

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

---

<!--
Below are template for copilot to generate CR message.
Please DO NOT modify it.
-->

### 🚀 Summary

copilot:summary

### 🔍 Walkthrough

copilot:walkthrough

fangsmile commented:
@mengxi-ream  What is the reason for this change?

mengxi-ream commented:
> @mengxi-ream What is the reason for this change?

The original implementation of `500px` has a type mismatch. The `height` parameter should be a number in accordance with the type declaration.

fangsmile commented:
![image](https://github.com/user-attachments/assets/1e77309b-8e27-404c-8b11-632e1af81196)
In fact, project's code logic support number|string, you can help modify this type to avoid `Type error` @mengxi-ream

mengxi-ream commented:
> ![image](https://private-user-images.githubusercontent.com/3468473/399210756-1e77309b-8e27-404c-8b11-632e1af81196.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzU2MDE5NTYsIm5iZiI6MTczNTYwMTY1NiwicGF0aCI6Ii8zNDY4NDczLzM5OTIxMDc1Ni0xZTc3MzA5Yi04ZTI3LTQwNGMtOGIxMS02MzJlMWFmODExOTYucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTIzMCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEyMzBUMjMzNDE2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ODNhYzM5NDM4NmEwMWFhZmE2ZGNlMmI4NWNjYzgwZDkxMzA3MGU1MDQ4ZmE5ODdmNTgwOTIyYmM2YjllNDRhYiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.RBHIo3Ow5YEK1QyqnqcuGshISDzVGgccSNocmq_TH5Q) In fact, project's code logic support number|string, you can help modify this type to avoid `Type error` @mengxi-ream

I have reverted the document changes and corrected the type declaration. @fangsmile
